operations/reservations.md

```diff
@@ -566,6 +567,53 @@ Extent of data to be returned. E.g. it is possible to specify that together with
 | `Id` | string | required | Unique identifier of the reservation group. |
 | `Name` | string | optional | Name of the reservation group. |
 
+#### Sex
```

Comment:
Is this intentional? I think it creates duplicates in  with objects in customers.md